src/Framework/Traits.cs

```diff
@@ -392,7 +392,9 @@ public bool EnableWarningOnCustomBuildEvent
 #if RUNTIME_TYPE_NETCORE
                     return true;
 #else
-                    return false;
+                    return ChangeWaves.AreFeaturesEnabled(ChangeWaves.Wave17_10) ?
+                        true :
+                        false;
```

Comment:
```suggestion
                    return ChangeWaves.AreFeaturesEnabled(ChangeWaves.Wave17_10);
```
.. or does it make it less readable/obvious?